apps/web/src/components/HoverCard/HoverCardCommunityInner.tsx

```diff
@@ -0,0 +1,193 @@
+import unescape from 'lodash/unescape';
+import Link from 'next/link';
+import { Route } from 'nextjs-routes';
+import { useMemo } from 'react';
+import { PreloadedQuery, usePreloadedQuery } from 'react-relay';
+import { graphql } from 'relay-runtime';
+import styled from 'styled-components';
+
+import InteractiveLink from '~/components/core/InteractiveLink/InteractiveLink';
+import Markdown from '~/components/core/Markdown/Markdown';
+import { HStack, VStack } from '~/components/core/Spacer/Stack';
+import { BaseM, BaseS, TitleM } from '~/components/core/Text/Text';
+import { HoverCardCommunityInnerQuery } from '~/generated/HoverCardCommunityInnerQuery.graphql';
+import { ErrorWithSentryMetadata } from '~/shared/errors/ErrorWithSentryMetadata';
+import { removeNullValues } from '~/shared/relay/removeNullValues';
+import handleCustomDisplayName from '~/utils/handleCustomDisplayName';
+
+import CommunityProfilePicture from '../ProfilePicture/CommunityProfilePicture';
+import { ProfilePictureStack } from '../ProfilePicture/ProfilePictureStack';
+
+export const HoverCardCommunityInnerQueryNode = graphql`
+  query HoverCardCommunityInnerQuery($communityAddress: ChainAddressInput!) {
+    communityByAddress(communityAddress: $communityAddress) {
+      __typename
+      ... on Community {
+        name
+        description
+        contractAddress {
+          address
+          chain
+        }
+        ...CommunityProfilePictureFragment
+        owners(onlyGalleryUsers: true, first: 10) {
```

Comment:
any reason we're using `first: 10`? maybe it can be fewer, since we display a smaller preview of holders?